src/node_options.cc

```diff
@@ -479,9 +479,9 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "show stack traces for the `throw` behind uncaught exceptions",
             &EnvironmentOptions::trace_uncaught,
             kAllowedInEnvironment);
-  AddOption("--trace-warnings",
```

Comment:
Removing this option is semver major. You may want to leave it in this file but ignore it elsewhere.